LabExT/Experiments/StandardExperiment.py

```diff
@@ -234,6 +234,7 @@ def run(self):
             data["measurement id long"] = measurement.id.hex
             data["instruments"] = measurement._get_data_from_all_instruments()
             data["measurement settings"] = {}
+            data["measurement_params"] = {name: param.value for name, param in measurement.parameters.items()}
```

Comment:
This information was meant to be stored under the `"measurement setting"` key, which is defined in the line above.

Originally, we mandated the person who writes the `Measurement` implementation to include saving of the measurement parameters into the `data` dict - see for example [the implementation in InsertionLossSweep](https://github.com/LabExT/LabExT/blob/8d4f1fa2fba6980ad149494aef8de12b522ddb6b/LabExT/Measurements/InsertionLossSweep.py#L146). The original reasoning for this was that the measurement code might actually change parameters (e.g. if a laser wavelength was chosen that the connected laser does not support) and then execute with the new parameters.

However, I think its good to re-think this mandate. I'd suggest you implement the measurement parameter copying from InsertionLossSweep here but save the params in the originally intended `"measurement settings"` key. In your plotting code, then use "measurement settings" instead of "measurement_params". Otherwise we have the same information stored twice in the output JSON, which is not nice.

Reply:
I saw the `"measurement setting"` key, however, I thought this might cause problems later, if some measurement implementations don't use the right structure or similar.

I think you're right though, the way you described is clearer. I will change it :)

Reply:
Done in 0731ff18d3a070c9afb71ce4ab3af912532bb684.